helper/tlsutil/generate.go

```diff
@@ -229,6 +229,21 @@ func keyID(raw interface{}) ([]byte, error) {
 	return kID[:], nil
 }
 
+// ParseCert parses the x509 certificate from a PEM-encoded value.
+func ParseCert(pemValue string) (*x509.Certificate, error) {
+	// The _ result below is not an error but the remaining PEM bytes.
+	block, _ := pem.Decode([]byte(pemValue))
+	if block == nil {
+		return nil, fmt.Errorf("no PEM-encoded data found")
+	}
+
+	if block.Type != "CERTIFICATE" {
```

Comment:
I was going to suggest using a constant from `encoding/pem` here but apparently they don't have any defined there? Weird.